src/utils/Client.util.ts

```diff
@@ -0,0 +1,52 @@
+/*
+Copyright 2022 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import * as grpc from "@grpc/grpc-js";
+import { KeyValueType } from "../types/KeyValue.type";
+
+/**
+ * Converts a KeyValueType to a grpc.Metadata object.
+ * @param metadata key value pair of metadata
+ * @returns grpc.Metadata object
+ */
+export function getGRPCMetadata(metadata: KeyValueType): grpc.Metadata {
```

Comment:
Could we rename to `generateGRPCMetadata`? sounds more like a generate instead of a getter

Also add default metadata and param check?

Reply:
It's more of a "converter" than "generator". Shall we call it `GRPCMetadataFromKeyValueType` and `HTTPMetadataQueryParamFromKeyValuePairType`? https://stackoverflow.com/questions/15962550/function-name-convention-for-convert-foo-to-bar

This might be too verbose though. Thoughts?

Reply:
or `createGRPCMetadata`? in the end I think I might be fine with: `convertKeyValueToGRPCMetadata` verbosity, yet it's an issue, but I'd rather have it descriptive than confusing :D

Reply:
Thanks @XavierGeerinck, went with `createXXX` then.